pontoon/base/urls.py

```diff
@@ -75,10 +75,14 @@
         name='pontoon.profile'),
 
     # Terminology Search
-    url(r'^search/$',
+    url(r'^terminology/$',
         views.search,
         name='pontoon.search'),
 
+    # Legacy: Redirect to /terminology
+    url(r'^search/$',
+        RedirectView.as_view(url="/terminology/", permanent=True)),
```

Comment:
Use `pattern_name='pontoon.search'` instead of `url` to make this not rely on the specific URL. :D